components/TextFields/tests/snapshot/MDCBaseTextFieldSnapshotTests.m

```diff
@@ -182,4 +182,16 @@ - (void)testFloatingLabelWithCustomColorWhileEditing {
   [self validateTextField:textField];
 }
 
+- (void)testDisabledTextField {
```

Comment:
nit: next time lets separate the this snapshot test addition from the intrinsic size changes. We like small PRs.